app/lib/features/quiz_registration/presentation/pages/quiz_registration_page.dart

```diff
@@ -232,25 +252,72 @@ class _QuizRegistrationPageState extends State<QuizRegistrationPage> {
                     height: 40,
                   ),
                   const Text('Latihan yang pernah diikuti'),
-                  const SizedBox(
-                    height: 10,
-                  ),
-                  Container(
-                    height: MediaQuery.of(context).size.height - 300,
-                    width: double.infinity,
-                    padding: const EdgeInsets.symmetric(horizontal: 16),
-                    decoration: BoxDecoration(border: Border.all()),
-                    child: Column(
-                        mainAxisAlignment: MainAxisAlignment.center,
-                        children: [
-                          const Text(
-                            'Silahkan klik Tombol `Daftar Latihan Bebras` dibawah untuk memulai',
-                          ),
-                          const SizedBox(
-                            height: 20,
-                          ),
-                          quizCard()
-                        ]),
+                  BlocConsumer<QuizRegistrationCubit, QuizRegistrationState>(
+                    listener: (context, state) {
+                      if (state is QuizRegistrationSuccess) {
+                        context
+                            .read<QuizRegistrationCubit>()
+                            .fetchParticipantWeeklyQuiz();
+                      }
+                    },
+                    builder: (context, state) {
+                      if (state is QuizRegistrationLoading) {
+                        return const Center(
+                          child: CircularProgressIndicator(),
+                        );
+                      }
+                      return SizedBox(
+                        height: MediaQuery.of(context).size.height - 300,
+                        width: double.infinity,
+                        child: BlocConsumer<QuizRegistrationCubit,
+                            QuizRegistrationState>(
+                          listener: (context, state) {
+                            // TODO: implement listener
+                          },
+                          builder: (context, state) {
+                            if (state is GetParticipantWeeklyQuizSuccess) {
+                              return ListView(children: [
+                                const SizedBox(
+                                  height: 10,
+                                ),
+                                for (final quiz in state.weeklyQuizzes)
+                                  quizCard(
+                                    quiz.quiz_title,
+                                    quiz.attempts.isNotEmpty
+                                        ? quiz.attempts[quiz.attempts.length -
+                                            1]['start_at'] as String
+                                        : '-',
```

Comment:
```suggestion
                                        ? quiz.attempts[quiz.attempts.length -
                                            1]['start_at'].toString()
                                        : '-',
```

Reply:
fix because of this
<img width="343" alt="image" src="https://github.com/ia-toki/bebras-pandai/assets/1970083/3b1a245e-3590-4f1e-8411-7b184201c593">